backend/src/packages/files/libs/helpers/get-url.ts

```diff
@@ -0,0 +1,17 @@
+function generateUrl(
```

Comment:
Use arrow function. Also, the name of the file and the name of the function should be the same